pkg/agent/proxy/proxier.go

```diff
@@ -105,6 +109,7 @@ func (p *proxier) removeStaleServices() {
 			}
 		}
 		delete(p.serviceInstalledMap, svcPortName)
+		delete(p.endpointInstalledMap, svcPortName)
```

Comment:
@tnqn Yes.

Reply:
@weiqiangt is it still supposed to remove the endpointInstalledMap here given that you also do it in `removeStaleEndpoints`?

Reply:
@hongliangl please reply here so we have the context to discuss. 
Moving your reply here:
>> is it still supposed to remove the endpointInstalledMap here given that you also do it in removeStaleEndpoints?

> Here removes all endpoints related with a stale service. Function removeStaleEndpoints removes stale endpoints of a normal service.

@weiqiangt From my understanding of https://github.com/vmware-tanzu/antrea/pull/1815, you need the endpointInstalledMap to be there so that you can identify stale endpoints to delete, right? I think deleting it here cause openflow leak?

Reply:
OK, I see that in this version, endpoints are still removed in `removeStaleServices` so it makes sense to delete `p.endpointInstalledMap`. 
There will be a conflict between this PR and #1815. @weiqiangt could you cooperate with @hongliangl to see how to merge these PRs?

Reply:
Thanks, @tnqn. I think this PR does not have many changes on the `syncProxyRules` and its sub-calls. I think we can first merge #1815 and then rebase #1703 onto #1815 without this line.